### PR TITLE
FIX remove useless javascript

### PR DESCRIPTION
**Pull request by hregis**
fix for #13146

### CODE REVIEW

File: htdocs/accountancy/admin/account.php [hunk 1]
```diff
@@ -272,6 +272,7 @@
 	if ($search_pcgsubtype) $param .= '&search_pcgsubtype='.urlencode($search_pcgsubtype);
     if ($optioncss != '') $param .= '&optioncss='.$optioncss;
 
+    /*
```
Comment (eldy) [inline htdocs/accountancy/admin/account.php]: If you remove this, when you make a search on a field in the list, you will trigger the load of the chart of account.

Comment (hregis) [reply htdocs/accountancy/admin/account.php]: @eldy 
ok i see, i split the form and remove javascript

File: htdocs/accountancy/admin/account.php [hunk 2]
```diff
@@ -336,11 +306,22 @@
...
     print ajax_combobox("chartofaccounts");
     print '<input type="submit" class="button" name="change_chart" id="change_chart" value="'.dol_escape_htmltag($langs->trans("ChangeAndLoad")).'">';
-    print '<input type="hidden" name="valid_change_chart" id="valid_change_chart" value="0">';
+
+    print '</form>';
 
     print '<br>';
 	print '<br>';
 
+	print '<form method="POST" id="searchFormList" action="'.$_SERVER["PHP_SELF"].'">';
```
Comment (eldy) [inline htdocs/accountancy/admin/account.php]: By splitting the form, you break a lot of feature, for example, you lose the page selected of the limit on list that must be into the form (the print_barre_liste contains input field for form)
A better solution would be to replace the button "Change And Load" that has type "submit" with a simple "<a>" link and add javascript onClick on it to make a simple javascript redirect of the request using the id of chart of account from the select box as parameter (this is the only parameter we need).

Comment (hregis) [reply htdocs/accountancy/admin/account.php]: @eldy 
and if i add the same var of other form ? look my last fix

**Comment by eldy** [comment]
I think this 9f097b1a0894904ae84f0ee14b4a40d2ed94c120 fix the trouble.

**Pull request closed without merging**